### PR TITLE
6597: echo a sql query through terminus

### LINKED ISSUE #6597: Using WP-CLI On The Pantheon Platform Doc Update
Re: [Using WP-CLI On The Pantheon Platform](https://pantheon.io/docs/wp-cli)

Priority: Low

## Issue Description:

The WP-CLI example for running an SQL query on a WordPress site uses a parameter to pass the query. This is problematic on Pantheon, because the Pantheon sshd rejects many common characters needed in SQL queries, including `;` and `?`.

The Drush page uses I/O redirection on the SQL query example to avoid this problem.

## Suggested Resolution

Change the example to use I/O redirection instead of a parameter.

Existing example: `terminus wp $site.$env -- db query "SELECT * FROM wp_users WHERE ID=1"`

Suggested improvement: `echo "SELECT * FROM wp_users WHERE ID=1" | terminus wp $site.$env -- db query`

### PR DESCRIPTION
**Pull request by EdwardAngert**
Closes #6597 

## Summary

**[Run SQL Queries Using WP-CLI on Pantheon](https://pantheon.io/docs/wp-cli#run-sql-queries-using-wp-cli-on-pantheon)** - To avoid the platform scrubbing characters that might be necessary for SQL commands (like `;` or `?`) in Terminus WP-CLI commands, use I/O redirection instead of a parameter.

## Effect

The following changes are already committed:

*
*

## Remaining Work

The following changes still need to be completed:

- [ ] List any outstanding work here

--------------------------------------------------
## Post Launch

**Do not remove** - To be completed by the docs team upon merge:
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-6658](https://pr-6658--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `52` | `55.00000000000001` | `3` |
| Best Practice Score | `79` | `79` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-6658` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/29916/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/29916/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-6658--pantheon-docs.my.pantheonfrontend.website/docs)

File: source/content/wp-cli.md [hunk 1]
```diff
@@ -66,10 +66,10 @@ Feeling comfortable with WP-CLI? Here are a [few of the many commands](https://d
 
 ## Run SQL Queries Using WP-CLI on Pantheon
 
-Use the `wp db query` command via [Terminus](/terminus) to run SQL queries against the database on Pantheon:
+Use `echo` to pipe (or _redirect_) a SQL query through WP-CLI via [Terminus](/terminus):
 
 ```bash{promptUser: user}
-terminus wp $site.$env -- db query "SELECT * FROM wp_users WHERE ID=1"
+echo "SELECT * FROM wp_users WHERE ID=1" | terminus wp $site.$env -- db query
```
Comment (greg-1-anderson) [inline source/content/wp-cli.md:72]: Since we are using `echo`, we should include the ending `;` at the end of the sql query, since we can.

**Review by greg-1-anderson — APPROVED** [review]
LGTM but I'd recommend adding the ending ; in `"SELECT * FROM wp_users WHERE ID=1;"`

File: source/content/wp-cli.md (hunk 1 above)
Comment (EdwardAngert) [reply source/content/wp-cli.md]: added!

**Review by joa-pan — APPROVED** [review]
LGTM

**Pull request merged**